Beta_Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -0,0 +1,1648 @@
+        'Contents': contents,
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': tableToMarkdown(title, contents, removeNull=True),
+        'EntryContext': context
+    })
+
+
+def create_exclusion_item_request(exclusion_type, exclusion_value, os_type, description, exclusion_mode,
+                                  path_exclusion_type, group_ids, site_ids):
+    endpoint_url = 'exclusions'
+
+    payload = {
+        "filter": {
+            "groupIds": group_ids,
+            "siteIds": site_ids
+        },
+        "data": {
+            "type": exclusion_type,
+            "value": exclusion_value,
+            "osType": os_type,
+            "description": description,
+            "mode": exclusion_mode
+        }
+    }
+
+    if path_exclusion_type:
+        payload['data']['pathExclusionType'] = path_exclusion_type
+
+    response = http_request('POST', endpoint_url, data=json.dumps(payload))
+    if response.get('errors'):
+        return_error(response.get('errors'))
+    if 'data' in response:
+        return response.get('data')[0]
+    return {}
+
+
+def get_static_indicators_command():
+    """
+    Get an export of all threat static indicators
+    """
+    # Init main vars
+    headers = ['Category ID', 'ID', 'Category Name', 'Description']
+    contents = []
+    context = {}
+    context_entries = []
+    title = ''
+
+    # Get arguments
+    limit = int(demisto.args().get('limit', 50))
+
+    # Make request and get raw response
+    static_indicators = get_static_indicators_request()
+    if limit:
+        static_indicators = static_indicators[:limit]
+
+    # Parse response into context & content entries
+    if static_indicators:
+        title = 'Sentinel One - Getting Static Indicators \n' + \
+                'Provides summary information and details for all threat static indicators.'
+        for static_indicator in static_indicators:
+            contents.append({
+                'ID': static_indicator.get('id'),
+                'Description': static_indicator.get('description'),
+                'Category Name': static_indicator.get('categoryName'),
+                'Category ID': static_indicator.get('categoryId')
+            })
+            context_entries.append({
+                'ID': static_indicator.get('id'),
+                'Description': static_indicator.get('description'),
+                'CategoryName': static_indicator.get('categoryName'),
+                'CategoryID': static_indicator.get('categoryId')
+            })
+
+        context['SentinelOne.Indicators(val.ID && val.ID === obj.ID)'] = context_entries
+
+    demisto.results({
+        'Type': entryTypes['note'],
+        'ContentsFormat': formats['json'],
+        'Contents': contents,
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': tableToMarkdown(title, contents, headers, removeNull=True),
+        'EntryContext': context
+    })
+
+
+def get_static_indicators_request():
```

Comment:
seems like this request is static, what happens if there are many results? will you get all results no matter what? isn't there a way to filter the results? (or atleast limit them)

Reply:
I'm removing this command since the data is irrelevant. Discussed with Maya about this.